hazelcast/src/main/java/com/hazelcast/wan/WanEventCounterContainer.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2008-2017, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.wan;
+
+import com.hazelcast.util.ConstructorFunction;
+
+import java.util.concurrent.ConcurrentHashMap;
+
+import static com.hazelcast.util.ConcurrencyUtil.getOrPutIfAbsent;
+
+/**
+ * Thread safe container for {@link WanEventCounter} grouped by service name.
+ */
+public class WanEventCounterContainer {
```

Comment:
can we put this class into `impl` package?

Reply:
Moved to `impl` package.